server/api/schema.ts

```diff
@@ -244,6 +246,8 @@ const typeDefs = gql`
             filterOptions: DonationFormFilterOptions
             sortBy: DonationFormSortOptions
         ): [DonationForm]
+
```

Comment:
nit: remove empty line so `unseenDonationFormsExist` is grouped with the donation form endpoints